benchmark.py

```diff
@@ -0,0 +1,148 @@
+import asyncio
+
+import pytest
+
+from async_lru import alru_cache
+
+try:
+    from pytest_codspeed import BenchmarkFixture
+except ImportError:  # pragma: no branch  # only hit in cibuildwheel
+    pytestmark = pytest.mark.skip("pytest-codspeed needs to be installed")
+else:
+    pytestmark = pytest.mark.benchmark
+
+
+EVENT_LOOP = asyncio.get_event_loop()
```

Comment:
Global event loops are problematic for several reasons:
- `asyncio.get_event_loop()` at module level is deprecated and will raise a DeprecationWarning in Python 3.10+ when there's no running loop
- A shared loop across tests can lead to state bleeding between tests
- pytest-asyncio creates its own event loop per test, making this global loop potentially unused or conflicting

Reply:
gotcha, reverting now

Reply:
reverted